resources/serverless/templates/_helper_tests.txt

```diff
@@ -1,6 +1,6 @@
 echo 'TESTING has started'
 sleep 20
-/app/main.test -test.v
+/app/main.test kyma-integration
```

Comment:
![image](https://user-images.githubusercontent.com/17271979/100079519-07001880-2e45-11eb-8f2c-5cc86a28f597.png)
Removing that flag will disable logging in time of running a test, why do it? If you want to pass some kind of option to the binary do it as an additional flag or by env